project-roles/trusted-committer.md

```diff
@@ -0,0 +1,162 @@
+# Title
+
+Trusted Comitter
+
+# Patlet
+
+Many inner-source projects will find themselves in a situation where
+they consistently receive feedback, features, and bug-fixes from a contributor.
+In these situations project maintainers seek ways to recognize and reward the
+work of the contributor above and beyond single contributions.
+
+# Context
+
+  - You are the maintainer of a cross-team library, service, or shared resource
+  - You receive regular contributions
+
+# Problem
+
+  - Project maintainers want to find ways to scale their ability to support a
+  project
+  - Project maintainers want to find ways to scale the longevity of the value
```

Comment:
That is a bit fuzzy to me (not a native english speaker, though). Are we saying that we want to make sure there's someone to take over the project in case the original maintainers move on (as mentioned in the forces section below)?

Reply:
I think that's one aspect of it, but another is making sure the library is continually supported. We've seen cases where great projects lose mindshare amongst engineers because there aren't enough people to triage issues or make sure bugs get fixed in a timely manner.

Reply:
Could transition to the community and not a team.

Reply:
Use simpler language.